<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manage</class>
 <widget class="QDialog" name="Manage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QComboBox" name="comboBox">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>25</height>
       </size>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBoxbook">
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Book</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lblname">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledname"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="lblauthor">
          <property name="text">
           <string>Author</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledauthor"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="lblpublishers">
          <property name="text">
           <string>Publishers</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledpublisher"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="lblsubject">
          <property name="text">
           <string>Subject</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledsubject"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="lblavailablebook">
          <property name="text">
           <string>Available Book</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxavailablebook">
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="lblborrowlist">
          <property name="text">
           <string>Borrow list</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWborrowlist"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="btneditborrow">
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnaddborrow">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btndeleteborrow">
            <property name="text">
             <string>delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QGroupBox" name="groupBoxuser">
     <property name="title">
      <string>User</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_23">
        <item>
         <widget class="QLabel" name="lblusername">
          <property name="text">
           <string>User Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledusername"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_22">
        <item>
         <widget class="QLabel" name="lblpassword">
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledpassword"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_21">
        <item>
         <widget class="QLabel" name="lblnewpassword">
          <property name="text">
           <string>New Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lednewpassword"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_20">
        <item>
         <widget class="QLabel" name="lblnewpasswordagain">
          <property name="text">
           <string>New Password Again</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lednewpasswordagain"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_25">
        <item>
         <widget class="QLabel" name="lblfullname">
          <property name="text">
           <string>Full Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledfullname"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="lblemailaddres">
          <property name="text">
           <string>Email Addres</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledemailaddres"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <item>
         <widget class="QLabel" name="lblpcode">
          <property name="text">
           <string>Pcode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledpcode"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="lblborrowlist_4">
          <property name="text">
           <string>Borrow list</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWborrowlist_4"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_24">
          <item>
           <widget class="QPushButton" name="btneditborrow_4">
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnaddborrow_4">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btndeleteborrow_4">
            <property name="text">
             <string>delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBoxgroup">
     <property name="title">
      <string>Group</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <widget class="QLabel" name="lblnamegrp">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ledname_3"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="lblgrpmember">
          <property name="text">
           <string>Group Member</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWgrpmember"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <item>
           <widget class="QPushButton" name="btneditgrpmem">
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnaddgrpmem">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btndeletegrpmem">
            <property name="text">
             <string>delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btndelete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnedit">
       <property name="text">
        <string>Edite</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnadd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btndiscard">
       <property name="text">
        <string>Discard</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnsave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
